src/wordpress/generator.py

```diff
@@ -72,9 +83,21 @@ def __repr__(self):
         return repr(self.wp_site)
 
     def run_wp_cli(self, command):
+        """
+        Execute a WP-CLI command
```

Comment:
I would mention that we use delegation here, and that the `run_wp_cli` effective command is within the config

Reply:
Modifié.